packages/wonder-blocks-core/components/with-ssr-placeholder.js

```diff
@@ -22,14 +22,27 @@ type Props = {|
      * the server-side renderer and the rehydration -- or it defeats
      * the purpose of using the WithSSRPlaceholder component.
      */
-    placeholder: ?(() => ?React.Node),
+    placeholder: ?() => ?React.Node,
 |};
 
 type State = {|
     mounted: boolean,
 |};
 
-const HasHadFirstRenderContext = React.createContext<boolean>(false);
+type RenderState = "root" | "initial" | "standard";
+/**
+ * This is the context that tracks who is doing what in our SSR component tree.
+ *
+ * root:
+ *   no one has instigated an initial SSR render so the component that sees
+ *   this is responsible for updating the value to 1
```

Comment:
I don't understand this comment.  Which value is being updated to 1?

Reply:
Ah, yeah. I need to clarify this. Great catch. Thanks